src/NativeScript/ObjC/Inheritance/ObjCClassBuilder.mm

```diff
@@ -21,6 +21,10 @@
 #include "Interop.h"
 #include "TNSFastEnumerationAdapter.h"
 
+@protocol TNSDerivedClass
```

Comment:
It's still used within the class builder and I can't figure out a substitution. It's not on a hot path there so I guess it's fine.